### ISSUE TITLE
-ag no longer works

### ISSUE THREAD

**Issue by matthiaskrgr** [nabijaczleweli/cargo-update#96]
Previously I was able to update all outdated packages via `cargo install-update -ag`, this no longer works as of the latest update.

````
error: The following required arguments were not provided:
    <PACKAGE>...
USAGE:
    cargo install-update <PACKAGE>... --all --git
For more information try --help
````

**Comment by nabijaczleweli** [comment]
Argh, I knew there was a case I forgot about. Anyway, check [current `master`](https://github.com/nabijaczleweli/cargo-update/commit/de29530d8bd610f9abd818833438b7979c83641f)?

**Comment by matthiaskrgr** [comment]
Fix confirmed, thanks :)

**Comment by nabijaczleweli** [comment]
Lovely, cheers!

**Comment by nabijaczleweli** [comment]
Released in [`v1.7.0`](https://github.com/nabijaczleweli/cargo-update/releases/tag/v1.7.0).
